simplq/src/main/java/me/simplq/controller/model/token/TokenDetailResponse.java

```diff
@@ -17,4 +18,18 @@
   final Long aheadCount;
   final Boolean notifiable;
   final Date tokenCreationTimestamp;
+
+  public static TokenDetailResponse fromEntity(Token token) {
+    return new TokenDetailResponse(
+        token.getTokenId(),
+        token.getName(),
+        token.getContactNumber(),
+        token.getTokenNumber(),
+        token.getStatus(),
+        token.getQueue().getQueueName(),
+        token.getQueue().getQueueId(),
+        token.getAheadCount(),
+        token.getNotifiable(),
+        token.getTokenCreationTimestamp());
+  }
```

Comment:
@chalx I think defining a static factory method like this helps, so then you don't have to use a builder. 

I think we should use builder where we need to optionally set fields, and this where we need to construct from a POJO.